### PR TITLE
whitespace checks .rst and .md files too.

### PR DESCRIPTION
**Pull request by Yash-Varshney**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Python prompts `>>>` can now be enabled or disabled for easy copying of code into interactive sessions.
+- whitespace check now checks .rst and .md files too.
```
Comment (pdxjohnny) [inline CHANGELOG.md:10]:
```suggestion
- Whitespace check now checks .rst and .md files too.
```

**Comment by pdxjohnny** [comment]
Great job @Yash-Varshney! This looks like its working perfectly.

Could you please fix the trailing whitepsace it detected in `./model/transformers/README.md` on line 6 in this PR so that when we merge the CI doesn't end up failing. Thanks!!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Python prompts `>>>` can now be enabled or disabled for easy copying of code into interactive sessions.
+<<<<<<< HEAD
```
Comment (pdxjohnny) [inline CHANGELOG.md:10]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Whitespace check now checks .rst and .md files too.
+=======
```
Comment (pdxjohnny) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - `GetMulti` operation which gets all Inputs of a given definition
 - Support for async generator operations
+>>>>>>> 422cdb3a3920005f5c244689bfce8294414da1ae
```
Comment (pdxjohnny) [inline CHANGELOG.md:15]:
```suggestion
```

File: model/transformers/README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ Currently, dffml_model_transformers supports HuggingFace Transformers tensorflow
 
 ## Documentation
 
-Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
+Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers.
```
Comment (pdxjohnny) [inline model/transformers/README.md:10]: You're change here added trailing whitespace. We need to remove the trailing whitespace on line 6

```suggestion
Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
```

File: model/transformers/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ Currently, dffml_model_transformers supports HuggingFace Transformers tensorflow
 
 ## Documentation
 
-Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
+Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
```
Comment (pdxjohnny) [inline model/transformers/README.md:10]: Adding a space at the end of this line causes a whitespace error. We need to remove the space at the end of line 6

```suggestion
Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
```

Comment (Yash-Varshney) [reply model/transformers/README.md]: whoops, I didn't know that invisible whitespace can cause a problem. ;)
Thanks

**Pull request merged**